pootle/apps/import_export/templates/browser/index.html

```diff
@@ -31,15 +32,19 @@
   <form action="" method="post" enctype="multipart/form-data"
     id="js-upload-form" class="upload-form">
     {% csrf_token %}
-    {% for field in upload_form %}
-      {{ field }}
-    {% endfor %}
-    <input type="submit" />
+    <div class="hide">{{ upload_form.file }}</div>
+    <span {% if not has_admin_access %}class="hide"{% endif %}>
+    {% blocktrans with user=upload_form.user_id %}
+      as {{ user }}
```

Comment:
hmm... I didn't think about it.
Is it ok if we have such string?
`<a>Upload translations</a> as {{user}}`